### PR TITLE
<todo react app>: <Entire Milestone>

### PR DESCRIPTION
**Pull request by Bestbynature**
## This pull request contains:

- [x] to-do list components
- [x] styles components
- [x] router components
- [x] layout component
- [x] linter checks

### CODE REVIEW

File: src/routes/Home.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import Header from "@/components/Header";
+import TodosLogic from "@/components/TodosLogic";
+
+const Home = () => {
+    return ( 
+        // <div className="wrapper">
```
Comment (Bennyjoez) [inline src/routes/Home.jsx]: Kindly remove the commented code.

File: src/components/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+                            to={link.path}
+                            // className={({isActive})=>
+                            // isActive ? 'active__class' : undefined
+                            // }
+                            >{link.text}
+                            </NavLink>
+                        </li>
+                    )
+                })}
+            </ul> */}
```
Comment (Bennyjoez) [inline src/components/Navbar.jsx]: Kindly remember to remove commented/unused code. This makes the app lighter. 😄

File: README.md [hunk 3]
```diff
@@ -110,22 +111,26 @@ Clone this repository to your desired folder:
...
+- Run `npx eslint "**/*.{js,jsx}"` on the root of your directory of your project.
+- Fix linter errors.
+- Run
+```sh
+npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+```
+- Copy [.stylelintrc.json](https://github.com/microverseinc/linters-config/blob/master/react-redux/.stylelintrc.json) to the root directory of your project.
+- Run `npx stylelint "**/*.{css,scss}"` on the root of your directory of your project.
+- Fix linter errors.
+
```
Comment (Bennyjoez) [inline README.md]: I think when the user runs npm install on the terminal, they get every required package installed and ready for use. Kindly simplify this section. 👍🏾

**Review by Bennyjoez — CHANGES_REQUESTED** [review]
## Status: Changes Required ♻️ 

Hello, @Bestbynature You have done a really good job at implementing the requirements. However, there are a few changes that if worked on could make the project stand out even more. Kindly review the comments to get an idea of the requested changes.

File: src/routes/Home.jsx (hunk 1 above)
Comment (Bestbynature) [reply src/routes/Home.jsx]: Thank you so much @Bennyjoez

File: src/components/Navbar.jsx (hunk 2 above)
Comment (Bestbynature) [reply src/components/Navbar.jsx]: Thank you so much @Bennyjoez

File: README.md (hunk 3 above)
Comment (Bestbynature) [reply README.md]: Thank you so much @Bennyjoez

**Comment by Bestbynature** [comment]
Thank you so much @Bennyjoez. The files have been updated as requested.

**Review by Bennyjoez — APPROVED** [review]
Hello @Bestbynature,

# ✔️ STATUS: APPROVED 🎊 
You have done all that was required of you and it's now time to merge it. Congratulations on your achievements so far. Good job. 👍🏾

**Comment by Bestbynature** [comment]
Thanks, @Bennyjoez

**Pull request closed without merging**